### DISCUSSION TITLE
postgres connector. Is long polling not supported yet?

### DISCUSSION THREAD

**Discussion by rinat9177** [ConduitIO/conduit#1365] [Connectors]
I see in the documentation 3 operating modes: auto, logrepl, long_polling. But when I try to use long_polling mode I catch the warning "long polling not supported yet, only snapshot is supported"

**Comment by hariso** [comment]
@rinat9177 Haris from the Conduit team here. Thanks for reaching out to us! I'll need to check the details of this with the team, but yeah, long polling CDC is not supported. I'll check if there's a special reason why this is still in the docs.

**Comment by hariso** [comment]
@rinat9177 We'll remove `long_polling` as a CDC option from docs and configs to avoid confusion. The implementation of long polling CDC will be tracked here: https://github.com/ConduitIO/conduit-connector-postgres/issues/150.

**Comment by raulb** [comment]
@rinat9177 Thank you for reporting this issue. We intended to implement this feature, but we made the mistake of not completing the configuration. I have just cleaned this up on this pull request https://github.com/ConduitIO/conduit-connector-postgres/pull/153. As @hariso mentioned, we will address this and track it in the issue at https://github.com/ConduitIO/conduit-connector-postgres/issues/150.

Thank you again!
